controllers/getArticleComments.js

```diff
@@ -0,0 +1,20 @@
+const { fetchArticleComments } = require("../models/fetchArticleComments");
+
+exports.getArticleComments = (req, res, next) => {
+  const { article_id } = req.params;
+
+  if (/^\d+$/.test(article_id) === false) {
+    return Promise.reject({ status: 400, msg: "bad request!" }).catch((err) => {
+      next(err);
+    });
+  } else {
```

Comment:
SQL is going to throw and error if we try to pass a string as a number, i would let sql handle this issue instead